### DISCUSSION TITLE
"finding Duplicates" syntax in Dbeaver

### DISCUSSION THREAD

**Discussion by majidhosseini346** [dbeaver/dbeaver#19039] [Q&A]
Hi- it might be simple and silly question as I am newbie in DBeaver tool. Does anyone know which syntax can be used to query on duplicate values? 
P.s. Like what we have in SQL (Group By) it seems it doesn't exist in DBeaver. Thanks

**Comment by LonwoLonwo** [comment]
Hello @majidhosseini346 

Maybe the Grouping panel will be helpful for you: https://github.com/dbeaver/dbeaver/wiki/Panels#grouping-panel
